yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -182,7 +193,7 @@ func gatherAssessmentDataFromPG() (err error) {
 	}
 
 	homebrewVoyagerDir := fmt.Sprintf("yb-voyager@%s", utils.YB_VOYAGER_VERSION)
-	gatherAssessmentDataScriptPath := filepath.Join("/", "etc", "yb-voyager", "gather-assessment-data", "postgresql", "yb-voyager-gather-assessment-data.sh")
+	gatherAssessmentDataScriptPath := filepath.Join("/", "etc", "yb-voyager", "gather-assessment-data", "postgresql", "yb-voyager-pg-gather-assessment-data.sh")
```

Comment:
When you know all the components of a path, there is no need of `filepath.Join()`. You can simply write it as a constant string:
```
gatherAssessmentDataScriptPath := "/etc/yb-voyager/gather-assessment-data/postgresql/yb-voyager-pg-gather-assessment-data.sh"
```